elements/pfe-icon/demo/pfe-tab.css

```diff
@@ -0,0 +1,461 @@
+  }
+  @at-root #{&}([pfe-variant="wind"][aria-selected="false"]:last-of-type) {
+    @include browser-query(ie11) {
+      border-bottom: 1px solid transparent;
+    }
+  }
+  @at-root #{&}([pfe-variant="wind"][aria-selected="false"]:last-of-type):hover {
+    @include browser-query(ie11) {
+      border-bottom: #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    }
+  }
+}
+
+:host{
+  @at-root #{&}([vertical][pfe-variant="wind"]){
+    --pfe-tabs__tab--BorderLeft: #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-var(surface--border)};
+    @include browser-query(ie11) {
+      border-left: #{pfe-fetch(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-fetch(surface--border)};
+    }
+  }
+  @at-root #{&}([pfe-variant="wind"][aria-selected="false"]:hover){
+    --pfe-tabs__tab--BorderBottom: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    @include browser-query(ie11) {
+      border-bottom:  #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    }
+  }
+  @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="true"]){
+    --pfe-tabs__tab--BorderRight: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} transparent;
+    --pfe-tabs__tab--BorderLeft:  #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(highlight)};
+    padding-left:                 calc(#{pfe-var(container-padding)} * 1.5 - 2px);
+    @include browser-query(ie11) {
+      border-right:  0;
+      border-left:   #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(highlight)};
+      padding-left:  10px;
+      border-bottom: 0 !important;
+    }
+  }
+  @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]){
+    --pfe-tabs__tab--BorderRight: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} transparent;
+  }
+  @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]:hover){
+    --pfe-tabs__tab--BorderRight: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} transparent;
+    --pfe-tabs__tab--BorderBottom: 0;
+    --pfe-tabs__tab--BorderLeft: #{pfe-local(BorderWidth--hover)} #{pfe-var(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    padding-left: calc(#{pfe-var(container-padding)} * 1.5 - 2px);
+    @include browser-query(ie11) {
+      border-right:  0;
+      border-bottom: 0;
+      padding-left:  10px;
+      border-left:   #{pfe-local(BorderWidth--hover)} #{pfe-fetch(surface--border-style)} #{pfe-local(BorderColor--hover)};
+    }
+  }
+}
+
+/// ===========================================================================
+/// WIND TAB VARIANTS / saturated
+/// ===========================================================================
+:host {
+  @at-root #{&}([on="saturated"][pfe-variant="wind"]) {
+    color: #{pfe-var(link--focus--on-dark)};
```

Comment:
@castastrophe do we want to override the value of the `--pfe-tabs--Color` local variable here, instead of redefining the color property?

Reply:
I think in general we actually want to move in this direction - redefining the property, `color`, for example, instead of updating the local variable, such as `--pfe-tabs--Color`.  This would allow `--pfe-tabs--Color` to remain empty in the component so that the variable hook continues to work, even if attributes are used.

Reply:
I totes see the value of that, but how do we decide how to do one vs the other? looking at the temp pfe-cta.css file in the master branch, I see the various attributes redefining the local variables:
![image](https://user-images.githubusercontent.com/456863/87824373-248ff680-c843-11ea-83b4-85f866c01f35.png)

Reply:
That's kind of what I'm saying. We're not doing this now (redefining the properties) but maybe we should be.